enterprise/backend/src/metabase_enterprise/semantic_search/task/indexer.clj

```diff
@@ -53,7 +53,7 @@
         (.interrupt execution-thread)))))
 
 (def ^:private ^Duration startup-delay (Duration/parse "PT10S"))
-(def ^:private ^Duration run-frequency (Duration/parse "PT10S"))
+(def ^:private ^Duration run-frequency (Duration/parse "PT20S"))
```

Comment:
Potentially controversial, but this felt like a better pacing to me. Our idle sleep time is 3s and the cold exit timeout is now 5s, so in the idle state we sleep 3s, sleep 3s, yield to quartz, sleep another 10-6 = 4s, then get rescheduled. So the drum beat was pretty steady, and since there was little difference between the idle sleep and the reschedule sleep, you might as well not yield control at that point (which could also be an option). With the new values, we sleep 3s, 3s, then 14s before getting rescheduled, which still feels fairly responsive.

Reply:
^ reverted the change to the cold exit timer, so now we sleep 10 x 3s, yield to quartz, then get immediately rescheduled.

Reply:
Yes I think this is a good change and a better default for now - I expected to balance all the related variables using the metrics (coming soon) on idle/busy times, indexing delays etc. The values that are in there are literally first guesses!